core/cli/src/lib.rs

```diff
@@ -19,6 +19,9 @@
 #![warn(missing_docs)]
 #![warn(unused_extern_crates)]
 
+#[macro_use(telemetry)]
+extern crate substrate_telemetry;
```

Comment:
I think in 2018 we should import macro by namespace: use substrate_telemetry::telemetry;

Reply:
If it is one macro, that could be done per import. If you need multiple macros like `info`, `error` etc, I would still vote for `macro_use`.

Reply:
I don't have strong opinion on it actually but I think we had converted every macro_use to namespace import, for example for decl_* macros : https://github.com/paritytech/substrate/blob/master/srml/example/src/lib.rs#L23